com.microsoft.mrtk.graphicstools.unity/Runtime/Utilities/DrawFullscreenFeature.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#if GT_USE_URP
+using UnityEngine;
+using UnityEngine.Rendering.Universal;
+
+namespace Microsoft.MixedReality.GraphicsTools
+{
+    public enum BufferType
+    {
+        CameraColor,
+        Custom 
+    }
+
+    /// <summary>
+    /// Forked from: https://github.com/Unity-Technologies/UniversalRenderingExamples/tree/master/Assets/Scripts/Runtime/RenderPasses
+    /// Performs fullscreen blit via a custom Render Pass
+    /// </summary>
+    public class DrawFullscreenFeature : ScriptableRendererFeature
+    {
+        [System.Serializable]
+        /// <summary>
+        /// Class <c>Settings</c> outlines controls for the Render Feature
+        /// </summary>
+        public class Settings
+        {
+            public RenderPassEvent renderPassEvent = RenderPassEvent.AfterRenderingOpaques;
+
+            public Material blitMaterial = null;
+            public string blitSourceTextureName = "_SourceTex";
+            public int blitMaterialPassIndex = -1;
+            public BufferType sourceType = BufferType.CameraColor;
+            public BufferType destinationType = BufferType.CameraColor;
+            public string sourceTextureId = "_SourceTexture";
+            public string destinationTextureId = "_DestinationTexture";
+            public bool restoreCameraColorTarget = true;
+        }
+
+        public Settings settings = new Settings();
+        private DrawFullscreenPass blitPass;
+        /// <summary>
```

Comment:
Nit: could you add a newline between the comment and the above variable or function? 
```suggestion

        /// <summary>
```